FusionIIIT/applications/academic_information/models.py

```diff
@@ -96,6 +102,7 @@ class Student(models.Model):
     father_name = models.CharField(max_length=40, default='')
     mother_name = models.CharField(max_length=40, default='')
     hall_no = models.IntegerField(default=0)
+    hall_id = models.CharField(max_length=255,null=True,blank=True, choices=Constants.HALL_NAMES)
```

Comment:
use hall_name

Reply:
Actually hall_id uses the data of hall_id in the hostel management hall table so we kept its name as hall_id only